components/atoms/Logo.vue

```diff
@@ -1,7 +1,5 @@
 <template>
-  <div>
-    <img :class="getSize" :src="src" />
-  </div>
+  <img :class="getSize" :src="src" />
```

Comment:
According to the Vue specification document, every component must have a single root element, and in this case, `<img>` it is, so I removed `<div>` element.

https://jp.vuejs.org/v2/guide/components.html#%E5%8D%98%E4%B8%80%E3%81%AE%E3%83%AB%E3%83%BC%E3%83%88%E8%A6%81%E7%B4%A0